pkg/api/config.go

```diff
@@ -1,9 +1,12 @@
 package api
 
 import (
+	"fmt"
 	"strings"
 	"time"
 
+	"github.com/getlantern/deepcopy"
```

Comment:
Nit: We could use kubes DeepCopy codegeneration here by extending https://github.com/openshift/ci-tools/blob/eb21567cf6c99a76bb707bdf263b4b3fb751cd51/hack/update-codegen.sh#L5 and enabling it on package level like here: https://github.com/openshift/ci-tools/blob/eb21567cf6c99a76bb707bdf263b4b3fb751cd51/pkg/api/testimagestreamtagimport/v1/doc.go#L1-L4

Just mentioning that because I had a couple of situations in the past where I thought that would be useful for this package

Reply:
Sure, let's give that a shot but I don't have much experience with it - if it just works then I'll happily include it here.

Reply:
Done. Had to disable for some types though.